src/ffi/methodobject.rs

```diff
@@ -8,6 +8,12 @@ extern "C" {
     pub static mut PyCFunction_Type: PyTypeObject;
 }
 
+#[cfg(Py_3_9)]
+#[inline]
+pub unsafe fn PyCFunction_CheckExact(op: *mut PyObject) -> c_int {
+    (Py_TYPE(op) == &mut PyCFunction_Type) as c_int
+}
+
 #[inline]
 pub unsafe fn PyCFunction_Check(op: *mut PyObject) -> c_int {
     (Py_TYPE(op) == &mut PyCFunction_Type) as c_int
```

Comment:
This definition `PyCFunction_Check` is the same as the new `PyCFunction_CheckExact` - does it need updating?

Reply:
Oops, thanks. Looks like this one was renamed in Python 3.9, with the new definition of `PyCFunction_Check` using `PyObject_TypeCheck`.

Reply:
Looks like 3.9 introduced the `Py_IS_TYPE` macro in `object.h`, which I'll add -- should help catch these changes when eyeballing